### DISCUSSION TITLE
打造精准回复的问答机器人的问题

### DISCUSSION THREAD

**Discussion by ILG2021** [ymcui/Chinese-LLaMA-Alpaca-2#557] [Q&A]
大语言模型自己生成的答案会存在“胡编”或者答非所问的现象，如何让模型准确的识别意图，然后原样复制答案回复，如果训练数据中没有的问题就回复不知道？因为我们的应用对答案的精确性要求比较高。不知道哪位大佬可以指导下？

我看到有用大语言模型做知识库的，是不是知识库更加适合我的需求，或者不使用大语言模型，只是用意图识别即可？

**Comment by qq614438892** [comment]
同问
